app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -292,7 +297,7 @@ class BloodSugarSummaryView(
   }
 
   private fun isBloodSugarEditable(measurement: BloodSugarMeasurement): Boolean {
-    if (bloodSugarSummaryConfig.bloodSugarEditFeatureEnabled.not() || measurement.reading.type is Unknown) {
+    if (features.isDisabled(EditBloodSugar) || measurement.reading.type is Unknown) {
```

Comment:
Do we even need a feature flag for `EditBloodSugar` anymore since this is enabled locally?

Reply:
We don't. Let's remove this later.